pkg/alerts/nrql_conditions_integration_test.go

```diff
@@ -712,3 +712,87 @@ func TestIntegrationNrqlConditions_StreamingMethods(t *testing.T) {
 		}
 	}()
 }
+
+func TestIntegrationNrqlConditions_IgnoreOnExpectedTermination(t *testing.T) {
+	t.Parallel()
+
+	testAccountID, err := mock.GetTestAccountID()
+	if err != nil {
+		t.Skipf("%s", err)
+	}
+
+	var (
+		randStr                     = mock.RandSeq(5)
+		conditionName      = fmt.Sprintf("test-nrql-condition-%s", randStr)
+		conditionInput = NrqlConditionCreateInput{
+			NrqlConditionCreateBase: NrqlConditionCreateBase{
+				Description: "test description",
+				Enabled:     true,
+				Name:        conditionName,
+				Nrql: NrqlConditionCreateQuery{
+					Query:            "SELECT rate(sum(apm.service.cpu.usertime.utilization), 1 second) * 100 as cpuUsage FROM Metric WHERE appName like 'Dummy App'",
+					EvaluationOffset: &nrqlConditionBaseEvalOffset,
+				},
+				RunbookURL: "test.com",
+				Terms: []NrqlConditionTerm{
+					{
+						Threshold:            &nrqlConditionBaseThresholdZeroValue,
+						ThresholdOccurrences: ThresholdOccurrences.AtLeastOnce,
+						ThresholdDuration:    600,
+						Operator:             AlertsNRQLConditionTermsOperatorTypes.ABOVE,
+						Priority:             NrqlConditionPriorities.Critical,
+					},
+				},
+				Expiration: &AlertsNrqlConditionExpiration{
+					CloseViolationsOnExpiration: true,
+					ExpirationDuration:          &nrqlConditionBaseExpirationDuration,
+					OpenViolationOnExpiration:   false,
+					IgnoreOnExpectedTermination: true,
+				},
+				ViolationTimeLimitSeconds: 3600,
+			},
+		}
+		updateInput = NrqlConditionUpdateInput{
+			NrqlConditionUpdateBase: NrqlConditionUpdateBase{
+				Expiration: &AlertsNrqlConditionExpiration{
+					CloseViolationsOnExpiration: true,
+					ExpirationDuration:          &nrqlConditionBaseExpirationDuration,
+					OpenViolationOnExpiration:   false,
+					IgnoreOnExpectedTermination: false,
+				},
+			},
+
+		}
+	)
+
+	// Setup
+	client := newIntegrationTestClient(t)
+	testPolicy := AlertsPolicyInput{
+		IncidentPreference: AlertsIncidentPreferenceTypes.PER_POLICY,
+		Name:               fmt.Sprintf("test-alert-policy-%s", randStr),
+	}
+	policy, err := client.CreatePolicyMutation(testAccountID, testPolicy)
+	require.NoError(t, err)
+
+	// Test: Create (static condition with streaming methods fields)
```

Comment:
Nit: Update comment to `Test: Create (static condition with expected termination field)